Makefile

```diff
@@ -56,6 +56,7 @@ lint:
 	  --global module \
 	  --global require \
 	  --global self \
+	  --rule 'indent: [off]' \
```

Comment:
As you know, I'm a big fan of linting. As soon as a rule forces me to do something I don't want to do, though, I'm inclined to disable it. I can police formatting during reviews. :)

Please remove the existing occurrence of `/* eslint-disable indent */` in whichever commit adds the `--rule`.